pom.xml

```diff
@@ -81,6 +81,22 @@
     <!-- Libraries -->
     <dependencies>
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.8.2</version>
```

Comment:
if jackson-* actually turn out to be necessary, please externalize this version number to a property like jackson.version